docker/Dockerfile.alpine

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.14.3
+LABEL maintainer="Vincenzo Palazzo (@vincenzopalazzo) vincenzopalazzodev@gmail.com"
+
+
+WORKDIR /build
+
+RUN apk update && \
+    apk add ca-certificates alpine-sdk autoconf automake git libtool \
+    gmp-dev sqlite-dev python3 py3-mako net-tools zlib-dev libsodium gettext su-exec \
+    python3 py3-pip && \
+    apk add --upgrade fortify-headers
+
+RUN mkdir lightning
+COPY . lightning
+
+RUN cd lightning && \
+    # hack to address this issue https://github.com/ElementsProject/lightning/issues/4918
+    #rm -r cli/test/*.c && \
+    ./configure && git submodule update --init --recursive && \
+    pip3 install mrkd mistune==0.8.4 && \
+    make -j$(nproc) && \
+    make install
+
+RUN apk --update --allow-untrusted --repository http://dl-4.alpinelinux.org/alpine/edge/community/ add \
+      tor torsocks && \
+    apk --update --allow-untrusted --repository http://dl-4.alpinelinux.org/alpine/edge/testing/ add \
+      obfs4proxy
```

Comment:
What are these used for? I thought this was intended to be a builder for CI, not the image used in production by users.

Reply:
Just a copy and paste error, for now, this is only used to test the compilation process, so we don't need this